docs/area-owners.md

```diff
@@ -73,9 +73,9 @@ Note: Editing this file doesn't update the mapping used by `@msftbot` for area-s
 | area-System.Console                            | @jeffhandley         | @dotnet/area-system-console                         |                                                                                                                                                                                                                                                                                           |
-| area-System.Data                               | @sammonort          | @ajcvickers @davoudeshtehari @david-engel @roji     | <ul><li>Odbc, OleDb - @saurabh500</li></ul>                                                                                                                                                                                                                                               |
-| area-System.Data.Odbc                          | @sammonort          | @ajcvickers @roji                                   |                                                                                                                                                                                                                                                                                           |
-| area-System.Data.OleDB                         | @sammonort          | @ajcvickers @roji                                   |                                                                                                                                                                                                                                                                                           |
+| area-System.Data                               | @sammonort          | @ajcvickers @davoudeshtehari @david-engel @roji     | <ul><li>Odbc, OleDb - @saurabh500</li></ul>                                                                                                                                                                                                                                                |
+| area-System.Data.Odbc                          | @sammonort          | @ajcvickers @roji                                   |                                                                                                                                                                                                                                                                                            |
+| area-System.Data.OleDB                         | @sammonort          | @ajcvickers @roji                                   |                                                                                                                                                                                                                                                                                            |
```

Comment:
"Fix whitespace" seems to have deleted spaces out of 3 these lines unnecessarily?

Reply:
It was intentional, the vertical bars in these three lines were out of alignment.